pypy.py

```
@@ -0,0 +1,93 @@
+def append_to_list(value, my_list=[]):
+    my_list.append(value)  # Mutates the default list
+    return my_list
+
+print(append_to_list(1))
+print(append_to_list(2))  # Unexpected behavior, shares the same list
+
+# Global variable usage
+global_var = 10
+
+def modify_global():
+    global_var = 20  # Creates a new local variable instead of modifying the global
```

Ruff (0.14.0)

Local variable `global_var` is assigned to but never used

Remove assignment to unused variable `global_var`

(F841)